Package.resolved

```diff
@@ -15,8 +15,8 @@
         "repositoryURL": "https://github.com/freshOS/Stevia",
         "state": {
           "branch": null,
-          "revision": "87dd17a86240f16788239a78dd8be11c4b013150",
-          "version": "4.8.0"
+          "revision": "cfb1a1d2159277bb553c3dc46f3f742c0275566d",
+          "version": "5.1.2"
```

Comment:
Was this version update intentional?

Reply:
Yeah this was already done in creator/consumer app projects but not here.